README.md

```diff
@@ -93,14 +95,50 @@ Next, run the following commands from the Windows command line or Mac/Linux term
 
 ### 🚀 Launch
 
-Launch the application. Please chose a location where `docker-compose.yml` will be downloaded.
-
-```bash
-curl https://raw.githubusercontent.com/fastenhealth/fasten-onprem/refs/heads/main/docker-compose-prod.yml -o docker-compose.yml
-
-docker compose up -d
-```
-ℹ️ No local repository required.
+Launch the application. Please choose a location where `docker-compose.yml` and `set_env.sh` will be downloaded.
+
+To make your Fasten instance discoverable by companion mobile apps and other devices on your local network, you need to run the `set_env.sh` script before starting Docker Compose. This script sets the necessary `HOSTNAME` and `IP` values in a `.env` file, which is required for syncing.
+
+Here are the step-by-step instructions:
+
+1.  **Download necessary files:**
+    ```bash
+    curl https://raw.githubusercontent.com/fastenhealth/fasten-onprem/refs/heads/main/docker-compose-prod.yml -o docker-compose.yml
+    curl https://raw.githubusercontent.com/fastenhealth/fasten-onprem/refs/heads/main/set_env.sh -o set_env.sh
+    ```
+
+2.  **Prepare and run the environment setup script:**
+    Make the script executable and run it to generate your `.env` file. This will configure network variables required for Docker Compose.
+    ```bash
+    chmod +x ./set_env.sh
+    ./set_env.sh
+    ```
```

Comment:
really not a fan of this, unless the env variables it sets are completely optional,

Reply:
nevermind, I just saw the `Manual Configuration` section 😅